namespace/bcl.adoc

```diff
@@ -0,0 +1,46 @@
+= Binary Compatibility Levels
+
+This is a classification of the levels of binary compatibility support for Java EE 8 classic applications in terms of the level of difficulty/complexity of the associated implementation and tools.
+
+== Level 0 - Jakarta EE 8 Only
+* No transformation required
+* Requires a Jakarta EE 8 container
+* No mandated support for backwards compatibility
+** Up to implementors to decide what level of support to provide
+
+== Level 1 - Jakarta EE 8 Translation to EE 9
```

Comment:
a niggle about the ordering.  I see this as coming after your `Jakarta EE 9 With Mapping from EE 8` as a level as it belong to the variations were we no longer have an EE8 container.

Reply:
I see. My ordering was more about perceived complexity to implement vs a how much Java EE 8 existed at runtime.